frontend/src/layout/navigation/TopBar/AccountPopover.tsx

```diff
@@ -221,8 +223,10 @@ export function AccountPopoverOverlay(): JSX.Element {
             <AccountPopoverSection>
                 <ThemeSwitcher fullWidth type="tertiary" />
                 <LemonButton
-                    onClick={closeAccountPopover}
-                    to="https://posthog.com/changelog"
+                    onClick={() => {
+                        closeAccountPopover()
+                        openSidePanel(SidePanelTab.Docs, '/changelog')
+                    }}
```

Comment:
It's always keeping the `href` (aka `to`) in links, even if there's some custom in-app handling of the link: It's often frustrating to have de facto links which you can't right-click>"Open in new tab"

Reply:
Oh yeah good point